packages/nuxt-locokit/src/services/workspace.ts

```diff
@@ -0,0 +1,63 @@
+import { sdkClient } from './api'
+import { ITEMS_PER_PAGE_GROUPS } from './group'
+
+const ITEMS_PER_PAGE = 10
+
+export async function findWorkspaces(
+  {
+    params = {},
+    pageIndex = 0,
+    limit = ITEMS_PER_PAGE_GROUPS,
+    sort = {
+      createdAt: -1,
+    },
+  }: {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    params?: Record<string, any>
+    pageIndex?: number
+    limit?: number
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    sort?: Record<string, any>
+  } = {
+    params: {},
+    pageIndex: 0,
+    limit: ITEMS_PER_PAGE_GROUPS,
+    sort: {
+      createdAt: -1,
+    },
+  },
+) {
+  try {
+    return await sdkClient.service('workspace').find({
+      query: {
+        $limit: limit,
+        $skip: pageIndex * limit,
+        ...params,
+        // $sort: sort,
```

Comment:
Is this comment ok ?
This means we'll never be able to sort result ?

Reply:
It is commented because of this error : `must appear in the GROUP BY clause or be used in an aggregate` (cf previous MR).

Normally you have already fix this bug in your MR. (But I don't see tests about it)